scripts/hrmp-setup.ts

```diff
@@ -121,27 +151,64 @@ function printExtrinsic(name: string, extrinsic: SubmittableExtrinsic<"promise">
     console.log("");
 }
 
+async function maybeSubmitProposal(name: string, extrinsic: SubmittableExtrinsic<"promise">, endpoint: string, api: ApiPromise, shouldSubmit: boolean) {
+    printExtrinsic(name, extrinsic, endpoint);
+
+    console.log("Please check the printed extrinsic and enter the seed phrase to submit the proposal.");
+    if (shouldSubmit) {
+        let rl = readline.createInterface({
+            input: process.stdin,
+            output: process.stdout
+        });
+        const it = rl[Symbol.asyncIterator]();
+        const seed = await it.next();
```

Comment:
I think it would be more secure to have the user set the seed in the environment than pass it through stdin, but if this isn't for prod it's alright.

Reply:
Would it? I figured stdin would be more secure since it wouldn't be saved in bash history